pkg/deployer/deployer.go

```diff
@@ -58,7 +58,7 @@ func (e ethereumDeployer) DeployContract(contractInfo *ContractInfo, containerIn
 		Password string   `json:"password"`
 		Args     []string `json:"args"`
 	}{
-		Provider: fmt.Sprintf("http://%s:%s", containerInfo.InternalAddress, portBindings[0].Port),
+		Provider: fmt.Sprintf("http://%s:%s", containerInfo.InternalAddress, "8545"),
```

Comment:
I think this is an important distinction and maybe our struct needs to have `internalPortmap` and `externalPortmap`